src/components/ShoppingCart.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+class ShoppingCart extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      items: []
+    }
+  }
+
+  render(){
+    const { removeItem } = this.props;
+    return (
+      <section>
+        <p>Shopping Cart</p>
+        <ul>
+          {
+            this.props.cards && this.props.cards.map((el,idx) => (
```

Comment:
`map` applied to empty array returns an empty array. Also, `this.props.cards` is always an array and it always returns `true`. Remember, array which casted as boolean returns true. This checking is useless.
```suggestion
            this.props.cards.map((el, index) => (
```